manim/mobject/types/vectorized_mobject.py

```diff
@@ -934,6 +942,9 @@ def __init__(self, mapping_or_iterable={}, show_keys=False, **kwargs):
         self.submob_dict = {}
         self.add(mapping_or_iterable)
 
+    def __repr__(self):
+        return __class__.__name__ + "(" + repr(self.submob_dict) + ")"
```

Comment:
The parenthesis seem to be extra, no? `repr(dict)` already prints curly braces.

Reply:
`repr(dict)` prints the curly braces, yes. The round brackets are for showing how the VDict would look when instantiated.
```python
VDict({'s': ..., 'c': ...})
```
According to [this](https://stackoverflow.com/a/1984177/10674866) StackOverflow answer, it is recommended to let repr return a string as to how it would look when instantiated using one of the possible ways of instantiation.

Reply:
Ah gotcha, thanks :)